src/server/server.ts

```diff
@@ -56,7 +58,7 @@ const io = new socketIO.Server(server,  { cors: {
 const PORT = process.env.PORT || 3000;
 
 mongoose
-  .connect(`${process.env.MONGO_URI}`)
+.connect("mongodb://localhost:27017/real-time-chat-app")
```

Comment:
i might call this db "problox" because this is the main application db not just for real time chat app

Reply:
problox or problox-db would be better

Reply:
I second Bryce and Sirisha's observation. Consider changing the name of your db.

Reply:
Thanks for the suggestion. Will change in the future PR.